siriuspy/siriuspy/devices/fofb.py

```diff
@@ -649,6 +669,29 @@ def cmd_config_bpm_trigs(self, timeout=DEF_TIMEOUT):
             return False
         return True
 
+    def check_sysid_exc_disabled(self):
+        """Check whether SYSID excitation is disabled."""
+        if not self.connected:
+            return False
+        for ctl in self._ctl_refs.values():
+            if ctl.sysid_fofbacc_exc_state or ctl.sysid_bpm_exc_state:
+                return False
+        return True
+
+    def cmd_dsbl_sysid_exc(self, timeout=DEF_TIMEOUT):
+        """Command to disable SYSID excitation."""
+        devs = list(self._ctl_refs.values())
+        self._set_devices_propty(devs, 'SYSIDPRBSFOFBAccEn-Sel', 0)
+        if not self._wait_devices_propty(
+                devs, 'SYSIDPRBSFOFBAccEn-Sts', 0, timeout=timeout/2):
+            return False
+        self._set_devices_propty(devs, 'SYSIDPRBSBPMPosEn-Sel', 0)
+        if not self._wait_devices_propty(
+                devs, 'SYSIDPRBSBPMPosEn-Sts', 0, timeout=timeout/2):
+            return False
+        self._evt_fofb.cmd_external_trigger()
```

Comment:
Talking to @ericonr offline, we agree that it could be a feature implemented in the HLTiming device class, maybe a method with the event to be sent, the trigger(s) whose devices the method would check whether the EvtCnt PVs have been incremented, and a timeout to wait for the event to be delivered.